Slim/Player/Playlist.pm

```diff
@@ -84,12 +84,12 @@ sub track {
 		$objOrUrl = ${playList($client)}[$index];
 	}
 
-	if ( $objOrUrl && ($refresh || !blessed($objOrUrl)) ) {
+	if ( $objOrUrl && ($refresh || !blessed($objOrUrl) || ref($objOrUrl) eq 'Slim::Schema::RemoteTrack' && $objOrUrl->url) ) {
 
 		$objOrUrl = Slim::Schema->objectForUrl({
 			'url'      => $objOrUrl,
-			'create'   => 1,
-			'readTags' => 1,
+			'create'   => 0,
+			'readTags' => 0,
```

Comment:
Why this change?

Reply:
But doesn't the `$refresh` there mean we _want_ the code to read the tags to refresh them if needed? Did you test all those occurrences, eg. the below `makeVolatile()` where we _want_ to create new objects for formerly local tracks, as they wouldn't exist yet as remote tracks?

Reply:
Oops... you didn't change the `makeVolatile()` call. I take that back.